cmd/alertmanager/main.go

```diff
@@ -181,6 +182,9 @@ func buildReceiverIntegrations(nc *config.Receiver, tmpl *template.Template, log
 	for i, c := range nc.WebexConfigs {
 		add("webex", i, c, func(l log.Logger) (notify.Notifier, error) { return webex.New(c, tmpl, l) })
 	}
+	for i, c := range nc.WeComRobotConfigs {
```

Comment:
Leave a blank line to facilitate merging with other patches or upgrade versions.